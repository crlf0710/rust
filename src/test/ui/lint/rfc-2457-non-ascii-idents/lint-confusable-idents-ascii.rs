#![deny(confusable_idents_ascii)]
#![allow(confusable_idents)]

fn main() {
    let s1 = "rust";
    let sl = "rust2";  //~ ERROR identifier pair considered confusable
}
