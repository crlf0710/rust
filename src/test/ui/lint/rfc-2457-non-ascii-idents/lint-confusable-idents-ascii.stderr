error: identifier pair considered confusable between `s1` and `sl`
  --> $DIR/lint-confusable-idents-ascii.rs:6:9
   |
LL |     let s1 = "rust";
   |         -- this is where the previous identifier occurred
LL |     let sl = "rust2";
   |         ^^
   |
note: the lint level is defined here
  --> $DIR/lint-confusable-idents-ascii.rs:1:9
   |
LL | #![deny(confusable_idents_ascii)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

